modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -105,6 +106,7 @@ public ObjectDefinitionDeployerImpl(
 		ModelSearchRegistrarHelper modelSearchRegistrarHelper,
 		ObjectDefinitionLocalService objectDefinitionLocalService,
 		ObjectEntryLocalService objectEntryLocalService,
+		ObjectEntryService objectEntryService,
 		ObjectEntryManagerTracker objectEntryManagerTracker,
```

Comment:
@marcelabc I think the `objectEntryService` should come after the `objectEntryManagerTracker`